src/js/variant-widget/eva-variant-widget.js

```diff
@@ -479,11 +479,27 @@ EvaVariantWidget.prototype = {
             }
         };
 
+        var exportVCFButton = {
+            xtype: 'button',
+            text: 'Export as VCF <span class="badge" style="border-radius:30%;padding:0.1em;font-size:10px">beta</span>',
+            style: {
+                borderStyle: 'solid'
+            },            
+            listeners: {
+                click: {
+                    fn: function () {
+                        var url = EVA_VCF_DUMPER_HOST + '/' + EVA_VERSION + '/variants/headers?species=' +  _this.values.species + '&studies=' + _this.values.studies;
```

Comment:
this only writes the header, use something like https://wwwdev.ebi.ac.uk/eva/webservices/vcf-dumper/v1/v1/segments/X%3A10000000-11000000/variants?species=agambiae_agamp4&studies=PRJEB7894 for the search in  https://wwwdev.ebi.ac.uk/eva/EVA-1113/?Variant-Browser&species=agambiae_agamp4&selectFilter=region&studies=PRJEB7894&region=X%3A10000000-11000000&annot-vep-version=78&annot-vep-cache-version=26

Reply:
oops, the url has "v1" twice. the code in eva-web should put v1 only once